packages/docs/docs/02-Guides/01-Connecting/09-Tenderly fork.mdx

```diff
@@ -0,0 +1,46 @@
+# Tenderly fork
```

Comment:
Consider naming it `Mainnet forking`.
The idea is to fork Mainnet, and tenderly is one of the tools to achievie it